Toolset/palettes/tutorial/revtutorial.livecodescript

```diff
@@ -482,20 +482,20 @@ on revTutorialParseMenuHighlight pTokens, pStart, @rData
    put 0 into tSubMenuTry
    repeat while the number of elements in pTokens > \
          tMenuSegmentCount +  tSubMenuSegmentCount * tSubMenuTry
-      put "highlight " into tExpressionTry
-      if tSubMenuTry is not 0 then
-         put tSubMenuExpressionTry after tExpressionTry
-      end if
-      put tMenuString after tExpressionTry
-      revTutorialParseSubexpression tExpressionTry, pTokens, pStart, rData
-      if the result is empty then
-         return empty
-      end if
-      put tSubMenuString & " " after tSubMenuExpressionTry
-      add 1 to tSubMenuTry
-   end repeat
-   
-   return "Expected: highlight [menu item <token> of...] menu <token>"
+   put "highlight " into tExpressionTry
```

Comment:
@livecodeali what's happened to the indenting here? Is this a script editor issue?